### PR TITLE
Fix UG for clarification on pay, owe and settle

### LINKED ISSUE #301: Clarify on pay, owe and settle in UG

### PR DESCRIPTION
**Pull request by TY1Fan**
closes #301 

Narrowed scope for target user to Singapore-based in description in UG

### CODE REVIEW

**Comment by TY1Fan** [comment]
> image Is there a typo here? $50 instead of $500.00
> 
> Also, it might be better if we explain to them beside the screenshot also. Points out $50 has been taken from `owe` to `pay` (please help change the English)

Thanks for spotting this. Ive corrected the example

**Review by DennieDan — APPROVED** [review]

File: docs/UserGuide.md [hunk 1]
```diff
@@ -279,7 +279,7 @@ Updates the amount of tuition fee paid by the specified student after a lesson.
 
 ### Recording unpaid tuition fee of a student: `owe`
 
-Updates the amount of tuition fee owed by a specified student after a lesson.
+Updates the amount of tuition fee owed by a specified student after a lesson. You should use the [`settle` command](#settling-outstanding-fees-from-students-settle) to settle the tuition fee owed by the student after he/she pay for the outstanding fees.
```
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:282]: Not sure if we need to tell about settle in owe since functionalities are very different

**Review by Soumyadip-cmd — COMMENTED** [review]
Just some considerations

File: docs/UserGuide.md (hunk 1 above)
Comment (TY1Fan) [reply docs/UserGuide.md:292]: I was thinking like settle updates both owe and pay. Thats why I placed this line for owe also

File: docs/UserGuide.md [hunk 2]
```diff
@@ -251,7 +251,7 @@ Finds students whose names contain any of the given keywords *and* their tuition
 
 ### Receiving payment from a student: `pay`
 
-Updates the amount of tuition fee paid by the specified student after a lesson.
+Updates the amount of tuition fee paid by the specified student after a lesson. You should use the [`settle` command](#settling-outstanding-fees-from-students-settle) to settle the tuition fee owed by the student after he/she pay for the outstanding fees.
```
[the remark this answers is not in the document]
Comment (TY1Fan) [reply docs/UserGuide.md:264]: im not sure how changing to right after a lesson will clarify the diff between settle and pay

File: docs/UserGuide.md (hunk 1 above)
Comment (Soumyadip-cmd) [reply docs/UserGuide.md:292]: Alright then makes sense.

File: docs/UserGuide.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Soumyadip-cmd) [reply docs/UserGuide.md:264]: Alright then makes sense.

**Review by Soumyadip-cmd — APPROVED** [review]
LGTM

**Pull request merged**